py/src/ai/h2o/sparkling/ml/features/H2OWord2Vec.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from ai.h2o.sparkling import Initializer
+from ai.h2o.sparkling.ml.H2OStageBase import H2OStageBase
+from ai.h2o.sparkling.ml.Utils import Utils
+from ai.h2o.sparkling.ml.models import H2OMOJOModel
+from ai.h2o.sparkling.ml.params.H2OWord2VecParams import H2OWord2VecParams
+from pyspark import keyword_only
+from pyspark.ml.wrapper import JavaEstimator
+from pyspark.ml.param.shared import HasInputCol, HasOutputCol
+
+class H2OWord2Vec(HasInputCol, HasOutputCol, H2OWord2VecParams, H2OStageBase, JavaEstimator):
```

Comment:
Couldn't this class be generated?

Reply:
I had it generated in the previous attempt when it was based on H2OAlgorithm, but that attempt was discouraged. 

I do not like writing new generation logic just for this one instance. I would keep it like this

Reply:
After giving it another thought, the generated code will be really simple, I will generate the python class to avoid manually typing the parameters.

Reply:
Done